hutch_python/epics_arch.py

```diff
@@ -56,6 +77,8 @@ def create_arch_file(experiment, hutch=None, path=None, dry_run=False):
     dry_run : bool
         To indicate if only print to stdout the data that would be stored
         in the epicsArch file and not create the file.
+    update : bool
```

Comment:
DOC/MAINT: `update` isn't used in the function body here, instead we are currently always updating regardless of input. `cds_items` and `link` are both used, but neither is documented up here.